src/reducers/authorReducer.js

```diff
@@ -0,0 +1,21 @@
+import {
+  FETCH_AUTHORS_SUCCESS,
+  FETCH_AUTHORS_FAILURE
+} from '../actions/actionTypes';
+
+import initialState from '../store/initialState';
+
+const { authors } = initialState;
+
+const authorReducer = (state = authors, action) => {
+  const { type } = action;
+  switch (type) {
+    case FETCH_AUTHORS_SUCCESS:
+    case FETCH_AUTHORS_FAILURE:
+      return { results: action.results, errors: action.errors };
```

Comment:
it appears on line 15, that you didn't account for the `previous state` of the part of the store you are working on. This completely violates the principles of redux. Refer to Abramov's video: the creator of redux on how to update the store.
[https://egghead.io/lessons/react-redux-avoiding-object-mutations-with-object-assign-and-spread](url)

Reply:
fixed this